flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/json/SerializedValueDeserializer.java

```diff
@@ -34,6 +36,10 @@
 
 	private static final long serialVersionUID = 1L;
 
+	public SerializedValueDeserializer() {
```

Comment:
Why do we add this default constructor? I prefer to specify the type explicitly. Also if it is a simple `TypeReference<SerializedValue<Object>>`.

Reply:
this is because the `SerializedValueDeserializer ` would be used in a jackson annotation `@JsonDeserialize`

Reply:
Alright, this makes sense.